.github/workflows/python.yml

```diff
@@ -104,10 +108,17 @@ jobs:
           # Below will exit with non-zero status if model changes are missing migrations
           sentry django makemigrations -n ci_test --check --dry-run --no-input || (echo '::error::Error: Migration required -- to generate a migration, run `sentry django makemigrations -n <some_name>`' && exit 1)
 
-  test-py3:
-    name: 'py3 [ignore fails]'
+  # TODO(joshuarli):
+  # More jobs. Also, python2 equivalents as well.
+  # - kakfa+zookeeper devservices necessary for what test suites?
+  # - MIGRATIONS_TEST_MIGRATE=1
+  # - TEST_SUITE=relay-integration
+  # - TEST_SUITE=symbolicator
+  # - TEST_SUITE=cli
+
+  test-py3-simple-no-migrations:
+    name: '(required) python3.6 backend (simple, no migrations)'
```

Comment:
Is having `(required)` necessary? If we really want to make it required, it should be done at the GitHub branch enforcement level

Reply:
Similar to the previous `[ignore fails]`, this is just here to reduce confusion for now.